group_vars/exporters.yml

```diff
@@ -1,6 +1,8 @@
 ---
 blackbox_exporter_web_listen_address: "127.0.0.1:9115"
 
+node_exporter_textfile_dir: "/var/lib/node_exporter"
```

Comment:
This is necessary as role variables cannot be used on playbook level and we want to use it in https://github.com/prometheus/demo-site/blob/cec19d47117f717a69abee9c4bad78bcdddca673/playbooks/02_monitoring.yml#L76

Reply:
Ahh, makes sense.